modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -10,3 +10,5 @@ Review the following limitations before installing and using the AWS Load Balanc
 * The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller. As a result of disabling the IP traffic mode, the AWS Load Balancer Controller cannot use the pod readiness gate.
 
 * The AWS Load Balancer Operator adds command-line flags such as `--disable-ingress-class-annotation` and `--disable-ingress-group-name-annotation` to the AWS Load Balancer Controller. Therefore, the AWS Load Balancer Operator does not allow using the `kubernetes.io/ingress.class` and `alb.ingress.kubernetes.io/group.name` annotations in the `Ingress` resource.
+
+* The AWS Load Balancer Operator must be configured so the SVC is `NodePort` type instead of `LoadBalancer` type.
```

Comment:
```suggestion
* You have configured the AWS Load Balancer Operator so that the SVC type is `NodePort` and not `LoadBalancer`.
```

Reply:
Expand first occurence of AWS and SVC?

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

Reply:
Since the service types are ClusterIP, NodePort, and LoadBalancer, so maybe it is better to add the ClusterIP, so it could be the following, thanks.
`You have configured the AWS Load Balancer Operator so that the SVC type is `NodePort` and not `LoadBalancer` or `ClusterIP``

Reply:
Absolutely

Reply:
Done.